drivers/clock_control/clock_stm32_mco.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (C) 2024, Joakim Andersson
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+#include <stm32_ll_rcc.h>
+
+#define DT_DRV_COMPAT st_stm32_clock_mco
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(mco1), okay)
+#if DT_PROP(DT_NODELABEL(mco1), source) == NO_SEL
+	#define MCO1_SOURCE		LL_RCC_MCO1SOURCE_NOCLOCK
+#elif DT_PROP(DT_NODELABEL(mco1), source) == STM32_SRC_EXT_HSE
```

Comment:
Since sources may not be defined, I think all these checks should be of the form:
```suggestion
#elif defined(STM32_SRC_EXT_HSE) && DT_PROP(DT_NODELABEL(mco1), source) == STM32_SRC_EXT_HSE
```

Reply:
This part of the code was removed.